spec/service-discovery.md

```diff
@@ -459,36 +443,33 @@ Removes a federated service in the federated SM consumer.
-service). If `ServiceID` needs to be changed, the suggested approach would be to
-remove the entry and create a new one.
+Modifies any number of the fields of a federated service in the consumer platform, except `Name` (it it changed, it would be then considered to be a different
+federated service). If `Name` needs to be changed, the suggested approach would be to remove the federated service and create a new one with the new name. In an update message, the full object must be sent, meaning that the attributes that do not change, must be filled with their current values.
 
 **Request message parameters**
 
 | Parameter | Data type | Required | Description |
 | --- | --- | --- | --- |
-| `Name` | `string` | Yes | Human readable name for the service |
-| `FQDN` | `string` | Yes | FQDN the federated service exposes outside of the owner mesh |
-| `ServiceID` | `string` | Yes | Name assigned to the service in a particular owner SM. It must be unique in the owner SM and will be the value sent as the SNI header by the consumer SM to a particular owner SM. Each SM vendor will possibly have a different kind of SNI and thus this spec does not define a specific format. |
-| `SAN` | `string` | Yes | URI SAN of the Federated Service to enable end to end security, in SPIFFE format |
-| `MeshIngress` | `Array<Endpoint>` | Yes | The endpoints of the ingress of the mesh where the service endpoints can be reached |
-| `Protocols` | `Array<string>` | Yes | Protocols associated with the service endpoint |
-| `Description` | `string` | Yes | Description of the service |
-| `Tags` | `Array<string>` | Yes | Informative values for filtering purposes |
-| `Labels` | `Map<string, string>` | Yes | Informative array of KV for filtering purposes |
+| `Name` | `string` | Yes | The unique name that identifies the service within an owner platform, and serves as its unique ID. Uniqueness in the service owner platform. The consumer platform must take into account possible name collistions that may happen with other FederatedService consumed from other owner platforms through, for example, aliasing.|
+| `FQDN` | `string` | Yes | The FQDN (Fully Qualified Domain Name) the consumer platform uses as a local DNS name to access the imported FederatedService. The consumer platform DNS is programmed to reference the Endpoint of the imported FederatedService. It is unique within the service owner platform. The consumer platform must solve any possible naming collistions that may happen with other FederatedService consumed from other owner platforms through, for example, aliasing. |
+| `SANS` | `Array<string>` | Yes | List of URI SANs A list of SANs (Subject Alternative Names) to enable end-to-end security for the federated service. |
+| `Instances` | `Array<Instance>` | Yes | Represents an instance of the FederatedService. |
+| `Endpoints` | `Array<Endpoint>` | Yes | The endpoints that provide access to a FederatedService in the owner platform. For example, this can be an ingress. |
```

Comment:
```suggestion
| `Endpoints` | `Array<Endpoint>` | Yes | The endpoints that provide access to a `FederatedService` in the owner platform. For example, this can be an ingress. |
```